block/submit.go

```diff
@@ -128,11 +127,14 @@ func SubmitLoopInner(
 					}
 					return err
 				}
-				ticker.Reset(maxBatchSubmitTime)
-				pending = uint64(unsubmittedBlocksBytes())                                                                                 //nolint:gosec // bytes size is always positive
-				logger.Info("Submitted a batch to both sub-layers.", "n bytes consumed from pending", nConsumed, "pending after", pending) // TODO: debug level
+				pending = uint64(unsubmittedBlocksBytes()) //nolint:gosec // bytes size is always positive
+				if batchSkewTime() < maxSkewTime {
```

Comment:
can u elaborate on the use of this `if`?
why not call `trigger.Nudge()` after submission loop finished as before?